crates/postgres-subsystem/postgres-resolver/src/util.rs

```diff
@@ -81,18 +81,25 @@ async fn check_create_access<'a>(
     request_context: &'a RequestContext<'a>,
     input_context: Option<&'a Val>,
 ) -> Result<AbstractPredicate, PostgresExecutionError> {
-    Ok(subsystem
+    let access = subsystem
```

Comment:
Is there a need to introduce a variable here, since the other functions don't have one but just wrap the output of `subsystem.solve` in an `Ok`?